src/main/scala/com/algorithmia/handler/RequestHandler.scala

```diff
@@ -0,0 +1,25 @@
+package com.algorithmia.handler
+
+import play.api.libs.json._
+
+import scala.reflect.ClassTag
+import scala.util._
+
+case class RequestHandler[I]() {
+
+  def convertToInput(input: JsValue)(implicit s: Reads[I], obj: ClassTag[I]): Try[I] = {
+    Json.fromJson[I](input).asOpt match {
```

Comment:
this method still feels unnecessary... its basically mapping success to success, and failure to failure.

Reply:
it has a custom error message and I felt like the actual operation of converting from a JsValue to an I was significant enough to warrant it's own function. But I could see it living as a lambda in processRequest